Update tiup-cluster.md (#9334)
pingcap/docs: tiup/tiup-cluster.md

Deploy a TiDB cluster for production

Usage:
  tiup cluster [command]

Available Commands:
  check       Perform preflight checks for the cluster.
  deploy      Deploy a cluster for production
  start       Start a TiDB cluster
  stop        Stop a TiDB cluster
  restart     Restart a TiDB cluster
  scale-in    Scale in a TiDB cluster
  scale-out   Scale out a TiDB cluster
  destroy     Destroy a specified cluster
  clean       (EXPERIMENTAL) Cleanup a specified cluster
  upgrade     Upgrade a specified TiDB cluster
  display     Display information of a TiDB cluster
  list        List all clusters
  audit       Show audit log of cluster operation
  import      Import an exist TiDB cluster from TiDB-Ansible
  edit-config Edit TiDB cluster config
  show-config Show TiDB cluster config
  reload      Reload a TiDB cluster's config and restart if needed
  patch       Replace the remote package with a specified package and restart the service
  help        Help about any command

Flags:
  -c, --concurrency int     max number of parallel tasks allowed (default 5)

>       --format string       (EXPERIMENTAL) The format of output, available values are [default, json] (default "default")

Author: 回头我加一下中文

>       --format string       (EXPERIMENTAL) The format of output, available values are [default, json] (default "default")

Author: 中文那边已经加上了

  -h, --help                help for tiup
      --ssh string          (EXPERIMENTAL) The executor type: 'builtin', 'system', 'none'.
      --ssh-timeout uint    Timeout in seconds to connect host via SSH, ignored for operations that don't need an SSH connection. (default 5)
  -v, --version             version for tiup

>       --wait-timeout uint   Timeout in seconds to wait for an operation to complete, ignored for operations that don't fit. (default 120)

Reviewer: TThe second sentence is missing in the Chinese
```suggestion
      --wait-timeout uint   Timeout in seconds to wait for an operation to complete. Inapplicable operations are ignored. (defaults to `120`)
```

      --transfer-timeout int   Timeout in seconds when transferring PD and TiKV store leaders (default 300)

Global Flags:

>       --ssh string          (EXPERIMENTAL) The executor type: 'builtin', 'system', 'none'.

Reviewer: ```suggestion
      --ssh string          (Experimental) The executor type. Optional values are 'builtin', 'system', and 'none'.
```

      --transfer-timeout int   Timeout in seconds when transferring PD and TiKV store leaders (default 300)

Global Flags:

>       --ssh string          (EXPERIMENTAL) The executor type: 'builtin', 'system', 'none'.

Reviewer: ditto

  -r, --rename NAME        Rename the imported cluster to NAME

Global Flags:

>       --ssh string        (EXPERIMENTAL) The executor type: 'builtin', 'system', 'none'.

Reviewer: ditto

>       --ssh string        (EXPERIMENTAL) The executor type: 'builtin', 'system', 'none'.

Author: ```suggestion
      --ssh string        (Experimental) The executor type. Optional values are 'builtin', 'system', and 'none'.
```
